styleguide/styles.scss

```diff
@@ -103,36 +103,6 @@
   }
 }
 
-.cads-styleguide__breakpoint-label {
```

Comment:
`@storybook/addon-essentials` includes the docs addon but also a responsive breakpoint addon so switched to that in place of this breakpoint label.